plugin.py

```
@@ -464,28 +464,45 @@ def job(m: Movie) -> None:
+            )
+            LOGGER.info(
+                "Requested episode refresh for series_id=%s via account_id=%s",
+                series.id,
+                account_id
+            )
+            return True
+        else:
+            LOGGER.debug("Task '%s' not registered in Celery; skipping internal refresh.", task_name)
+            return False
+
     except Exception as e:  # pragma: no cover
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)